server/schedule/checker/rule_checker.go

```diff
@@ -81,6 +88,13 @@ func (c *RuleChecker) Check(region *core.RegionInfo) *operator.Operator {
 	return nil
 }
 
+func (c *RuleChecker) refreshOfflineCounter() {
+	// gc the offlineLeaderCounter if there no event more than 24 hours.
+	if len(c.offlineLeaderCounter) > 0 && time.Since(c.lastReplaceOpTime) > 24*time.Hour {
```

Comment:
Why 24 hours?

Reply:
Just for a long time. Offline is not triggered often.

Reply:
PTAL @rleungx